media/js/src/AssetDetail.jsx

```diff
@@ -29,7 +29,12 @@ export default class AssetDetail extends React.Component {
             selectionLayer: new VectorLayer({
                 source: new VectorSource()
             }),
-            selectedSelection: null,
+
+            // The player seems to work better when it's loaded initially
+            // as 'playing', and then paused immediately onReady.
+            // https://github.com/CookPete/react-player/issues/536#issuecomment-453869837
+            playing: true,
```

Comment:
In my dev env, when I click the video link from the collection, the video plays and does not pause.

Reply:
This has been addressed.